extensions/wikia/Wall/WallHelper.class.php

```diff
@@ -330,8 +329,7 @@ private function getCommentsData( $comments ) {
 			}
 
 			$items[$i]['author'] = $data['username'];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "username" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)